README.md

```diff
@@ -20,7 +20,14 @@ $ docker-compose run cihm-metadatabus bash
 To test a specific script, for example, dmdtask, run:
 
 ```
-$ docker-compose run cihm-metadatabus "./tests/testDmdtask.sh"
+$ tail -f log/root.log
+```
+
+Then, in another terminal, run:
+
+```
+$ docker-compose run cihm-metadatabus bash
```

Comment:
Mentioning the `bash` twice may be confusing when you read later.


In one terminal run the `docker-compose run cihm-metadatabus bash`

Then run whatever command you want, as often as you want.

It can even be an infinite loop in a single command.

```
russell@russell-XPS-13-7390:~/git/cihm-metadatabus$ docker-compose run cihm-metadatabus bash
Creating cihm-metadatabus_cihm-metadatabus_run ... done
tdr@6495e43707b5:~$ while :; do dmdtask ; sleep 1m ; done
```

Then in a separate terminal run tail.


```
russell@russell-XPS-13-7390:~/git/cihm-metadatabus$ tail -f log/root.log 
2022/09/12 18:20:26 - INFO {CIHM.TDR} [CIHM::Meta::Hammer2::hammer] Hammer2 skip=0 limit=9 maxprocs=4 timelimit=
2022/09/14 11:13:22 - INFO {CIHM.TDR} [CIHM::Meta::Dmdtask::dmdtask] Dmdtask
2022/09/14 11:13:31 - INFO {CIHM.TDR} [CIHM::Meta::Dmdtask::dmdtask] Dmdtask
2022/09/14 11:18:24 - INFO {CIHM.TDR} [CIHM::Meta::Dmdtask::dmdtask] Dmdtask
2022/09/14 11:19:06 - INFO {CIHM.TDR} [CIHM::Meta::Dmdtask::dmdtask] Dmdtask
2022/09/14 11:20:07 - INFO {CIHM.TDR} [CIHM::Meta::Dmdtask::dmdtask] Dmdtask
```